CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## main
 
 ### ✨ Features and improvements
+- remove custom `stringify` function in test [#668](https://github.com/maplibre/maplibre-style-spec/issues/668))
```

Comment:
I didn't think it's interesting enough to be in the changelog...

Reply:
OK, I will remove it.